examples/no_trusted_accounts.tf

```diff
@@ -0,0 +1,22 @@
+  module "terraform_state" {
+    source  = "StratusGrid/terraform-state-s3-bucket-centralized-with-roles/aws"
+    version = "2.0.0"
```

Comment:
This should be 3.0.0, and all filed should be updated as soo.

Reply:
@wesleykirklandsg Please see the updated examples and let me know if you see any mistakes.